doc/src/modules/printing.rst

```diff
@@ -307,31 +307,53 @@ This class implements Python printing. Usage::
     x = Symbol('x')
     e = 5*x**3 + sin(x)
 
-srepr
------
+Srepr
+-----------
```

Comment:
I found all the heading in capital in [printing docs](http://docs.sympy.org/0.7.6/modules/printing.html#module-sympy.printing.repr) yeah  I removed extra underlines.